cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/deploy/LocalApplicationManagerTest.java

```diff
@@ -61,26 +69,48 @@
 /**
  * Tests the functionality of Deploy Manager.
  */
+@RunWith(Parameterized.class)
 public class LocalApplicationManagerTest {
   @ClassRule
   public static final TemporaryFolder TMP_FOLDER = new TemporaryFolder();
   public static final Gson GSON = new Gson();
 
+  @Parameterized.Parameters
+  public static Collection<Object[]> parameters() {
+    return Arrays.asList(new Object[][] {
+      {true},
```

Comment:
Could you add one one more parameter to hold test name? It will be `@Parameterized.Parameters("{0}")` with first parameter as description. It would be much easier to see which test fails in case of failure

Reply:
Sure